tests/k256_auth.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "mode": 2,
+    "kem_id": 22,
+    "kdf_id": 1,
+    "aead_id": 1,
+    "info": "70aa544b76a9d75a2b98682243489b1a2a315cc2",
+    "ikmE": "4c74b4c2bf105ba4390c23399b43a0f08de95686133e90288deafcea786f313e",
+    "skEm": "be124a18ba7956629489da30493aae91a51ad2bd1a41f34b39ec6b28de946576",
+    "pkEm": "048f1100da3b5413c417e224262b45f146884e21691c0ffff11cd04a762598c5e739f8c5d460a328de39c94a1ab922c9419be89cc36d262cb7ce8a28f850f8e8ff",
+    "enc": "048f1100da3b5413c417e224262b45f146884e21691c0ffff11cd04a762598c5e739f8c5d460a328de39c94a1ab922c9419be89cc36d262cb7ce8a28f850f8e8ff",
+    "ikmR": "f3dc9707eac8feb1a86c96279e23318fae9f3e2c04aca5ca9e2ace204488bd35",
+    "skRm": "040fc95447fff5a811321da69ff4655d185d58edef93453ab23dfb1be2f02702",
+    "pkRm": "04a3935d9f2ea9c4b23cdf49f4761625b2acbc1fc89532fe2c3af9d1b1c61b9f167f61ba6125d47151df26e2ecfa851bd79719c99ff354c9b9e9619f25cb6ba6d7",
+    "ikmS": "9fbf7fcf111cc65b6079290c65d0839396104f2dfd39ad34196a4b29d4122383",
+    "skSm": "040fc95447fff5a811321da69ff4655d185d58edef93453ab23dfb1be2f02702",
```

Comment:
I'm a little suspicious of these test vectors. The value here is the same as for `skRm` but with a differet `ikmR` value.

Reply:
Yeah, I noticed that too. They don't really seem complete. I actually emailed with the author of that update proposal so I'll drop him an email and ask about the vectors.

Reply:
Any news on this? I'm happy to get this in when we figured out what's up with the test vectors and can agree on a set that works.

Reply:
Unfortunately no update. I've been on holiday the last few days so haven't been working on it. I've emailed the original author of the IANA update (who submitted the test vectors) but I've not gotten a response on that.

Reply:
I've been out as well... How about generating a few new values with their implementation to see if they produce the same results? Maybe only the test vectors are broken. I'm fine using different values. I just want to make sure that the implementations agree.

Reply:
I've been in touch with the other dev who is also going to have a look at the values too. I'm flying today and tomorrow but I'll be looking into this properly next week.

Reply:
Hey I've spent the morning on this and from what I can tell, we have the `base` values all correct. (without any encryption or export checks). The `auth` values are don't have a set of sender keys. 

Unrelated to the current issues, why are the private keys and `enc` values are 64 byte strings instead of 32 byte strings in the test vectors?

Reply:
nevermind that last question. monday morning 🤦